addons/hr_expense/models/mail_thread.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+class MailThread(models.AbstractModel):
+    _inherit = 'mail.thread'
```

Comment:
This should be an override in `hr.expense.sheet` so that you don't need to check for model name below